alphagov/govwifi-dev-docs
# Update deployment instructions

### What

* Update deployment instructions to point to GovWifi Concourse. 
* Create section specifically for continuous delivery, outlining how CD works on GovWifi and pointing to the relevant repos.

### Why

GovWifi has migrated off of the multi-tenanted "Big Concourse" to a single-tenanted Concourse maintained by the team.

This needs to be reflected in the documentation.

Link to Trello card (if applicable): https://trello.com/c/Qtwh6Unr/1760-document-new-ci-cd-approach

### Testing

1. Check out the branch
2. Run `make serve`
3. Navigate to the localhost URL specified in the `make serve` output
4. Navigate to the updated and/or newly created pages
